apps/coa/index.js

```diff
@@ -196,7 +196,19 @@ module.exports = {
     '/check-answers': {
       behaviours: [Summary, ModifyChangeURL],
       sections: require('./sections/summary-data-sections'),
-      template: 'summary'
+      template: 'summary',
+      next: '/privacy-policy'
+    },
+    '/privacy-policy': {
+      behaviours: [Summary],
+      fields: ['privacy-check'],
+      sections: require('./sections/summary-data-sections'),
```

Comment:
no need for `behaviours` and `sections` on this step

Reply:
Removed this from the step for privacy-policy